extension/test/cleanup-ctp-project.js

```diff
@@ -0,0 +1,7 @@
+const ctpClientBuilder = require('../src/ctp/ctp-client')
+const iTSetUp = require('./integration/integration-test-set-up')
```

Comment:
```suggestion
const itSetup = require('./integration/integration-test-set-up')
```

Reply:
This Pr is only for documentation, not for code conventions. I will resolve those comments.